discourse-doctor

```diff
@@ -348,6 +350,8 @@ print_done() {
 initialize_log_file() {
   rm -f $LOG_FILE
   touch $LOG_FILE
+  # prevent others from reading sensitive content from the log
+  chmod 0600 $LOG_FILE
```

Comment:
Instead of `touch` + `chmod` we should do it in a single command with `install -m 0600 /dev/null $LOG_FILE`. Then you can remove the comment.